### DISCUSSION TITLE
Huge database record(60,000) is not loading. Connection timeout. I set as default value on connection string.

### DISCUSSION THREAD

**Discussion by AaronRiboy** [serenity-is/Serenity#6614] [Help]
<img width="378" alt="image" src="https://user-images.githubusercontent.com/29535650/207273261-0f154de6-3b3c-4152-87d7-6b7d83374f2a.png">

I have tried to set the connection timeout 30s , 0s and more .Nothing is working. Please help.Thank you

**Comment by dfaruque** [comment]
try restarting your database service.

**Reply by AaronRiboy** [reply]
tried .Still same bro =( hmmm

**Comment by edwardch** [comment]
Have a look at this old issue [Handling a table with ~4.72 million records #4399](https://github.com/serenity-is/Serenity/issues/4399),
and this discussion [Read a table with 300,000,000 million records #6095](https://github.com/serenity-is/Serenity/discussions/6095)
